File: ayesha/Recursion and Backtracking/Introduction to Recursion/print-increasing.cpp
#include <iostream>
using namespace std;

void printIncreasing(int n)
{
    // write your code here
    if (n == 0)
    {
        return;
    }
    printIncreasing(n - 1);
    cout << n << "\n";
}

int main()
{
    int n;
    cin >> n;
    printIncreasing(n);
}

File: V.Vennela/Basics of Programming/Recursion and Backtracking/Introduction To Recursion/PrintIncreasing.java
import java.io.*;
import java.util.*;

public class PrintIncreasing {

    public static void main(String[] args) throws Exception {
        // write your code here
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        printIncreasing(n);
    }

    public static void printIncreasing(int n){
         if(n==0)
            return ;
        
          printIncreasing(n-1); 
          System.out.println(n);
    }

}
